.github/workflows/updateCron.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           cache: 'pnpm'
-          node-version: 18
+          node-version: 18.18.2
```

Comment:
need to set the same to other jobs?

Reply:
if use tsx, it should fixed

Reply:
for now, only these used tsx:

root npm scripts
```
 "updateLPsAPR": "pnpm turbo run build --filter=@pancakeswap/farms && NODE_PATH=./apps/web/src tsx --tsconfig ./apps/web/tsconfig.json scripts/updateLPsAPR.ts",
    "updateAptosLPsAPR": "pnpm turbo run build --filter=@pancakeswap/aptos-swap-sdk && NODE_PATH=./apps/aptos/src tsx --tsconfig ./apps/aptos/tsconfig.json  scripts/updateAptosLpsAPR/index.ts",
    "updateMerkl": "NODE_PATH=./apps/web/src tsx --tsconfig ./apps/web/tsconfig.json scripts/updateMerkl/index.ts",
 ```
 
 farm pkg
 
 `    "build": "npx tsx scripts/build.ts",`

Reply:
farm have fixed on previous PR, only this file need to be changed

Reply:
i mean for other jobs within this file